src/Search/Numeric.js

```diff
@@ -61,24 +61,26 @@ function Numeric({ obj }) {
     <FilterBox title={display || id} isActive={isActive}>
       <Box id={`form-${id}`} as="form" onSubmit={(e) => e.preventDefault()}>
         <Flex>
-          <Box>
+          <Box sx={{ marginRight: 1 }}>
             <Input
               type="number"
               id="min"
               name="min"
               placeholder="min"
               defaultValue={min}
               onChange={(e) => update(e.target.value, max, unit)}
+              fontSize={0}
             />
           </Box>
-          <Box>
+          <Box width={units.length == 0 && 1 / 2} sx={{ marginRight: units.length > 0 && 1 }}>
```

Comment:
I reckon it might be easier to put a left margin on the max and unit boxes instead -- it should avoid the conditional expression. Also, for widths, I reckon putting `flex: '1 1 0%'` on all three boxes, will let each of them fill a third of the space automatically (again, removing the need for the condition).